ivy_tests/test_ivy/test_functional/test_core/test_creation.py

```diff
@@ -751,7 +746,6 @@ def test_triu(
     ),
     dtype=helpers.get_dtypes("numeric", full=False),
     size=helpers.ints(min_value=1, max_value=3),
```

Comment:
It seems to me that the way you are using `size` to test is not correct. `size` is an integer, then you convert it to a 1-element tuple, and then you iterate through it to produce the `x{i}` which in practice will always be a single `x0`.

Reply:
I just verified this and it turns out you're right. The documentation for `helpers.ints` is misleading though as it says this helper function draws an arbitrarily sized list of integers. Replacing with `helpers.lists` should do the job though,

Reply:
Tbh you don't even need to generate a size. You could just get a non-None shape and a bool variable that will select whether to unpack or not.

Reply:
Thanks for the suggestion. Have updated the tests now :)